pycheribuild/projects/disk_image.py

```diff
@@ -342,7 +342,12 @@ def add_from_mtree(
         if strip_binaries:
             # Try to shrink the size by stripping all elf binaries
             entry = self.mtree.get(mtree_path)
+            if entry is None:
```

Comment:
this change seems wrong

Reply:
It was failing the tests, returning None, when I ran it on my local machine (AArch64).